lib/config/fileconf.go

```diff
@@ -167,6 +167,8 @@ type SampleFlags struct {
 	AppURI string
 	// NodeLabels is list of labels in the format `foo=bar,baz=bax` to add to newly created nodes.
 	NodeLabels string
+	// JoinMethod is the method that will be used to join the cluster, either token, iam or ec2
```

Comment:
Your docs say `"ec2"|"iam"`, but here you also include `token` as a valid option. Which is correct?